amy/templates/dashboard/training_progress.html

```diff
@@ -75,38 +75,48 @@ <h2>Your progress towards becoming a Carpentries Instructor</h2>
       </td>
     </tr>
     <tr>
-      <th rowspan="2">2. Get Involved</th>
+      <th>2. Get Involved</th>
       <td>
         {% if progress_get_involved %}
           {% include 'includes/progress_trainee_view.html' with progress=progress_get_involved %}
           {% if progress_get_involved.state == "n" %}
-          <p>Please wait while we evaluate your submission.</p>
+          <p>Your submission will be evaluated within 7-10 days.</p>
+          <p>You may edit or delete your submission before it is evaluated. If you delete it, you will be allowed to create a new submission afterwards.</p>
+          {% endif %}
+          <p>Details of your submission are below.</p>
+          <p>
+            <strong>Activity:</strong> {{ progress_get_involved.involvement_type }}<br/>
+            <strong>Date:</strong> {% firstof progress_get_involved.date|date:'F j, Y' "No date provided" %}<br/>
+            <strong>URL:</strong> {% firstof progress_get_involved.url "No URL provided" %}<br/>
+            <strong>Notes:</strong> {% firstof progress_get_involved.trainee_notes "No notes provided" %}
+          </p>
+          {% if progress_get_involved.state == "n" %}
+              <p class="edit-object float-left">
+                <a href="{% url 'getinvolved_update' progress_get_involved.pk %}" class="btn btn-primary">Edit</a>  
+              </p>
+              <div class="delete-object float-left" style="padding-left: 2%;">
```

Comment:
Can you use [`.p`](https://getbootstrap.com/docs/4.6/utilities/spacing/#notation)-style class here? What I'm mostly concerned about is the percentage, as I think we're not using this unit for spacing anymore.